src/main/java/org/jabref/model/entry/Langid.java

```diff
@@ -0,0 +1,90 @@
+package org.jabref.model.entry;
+
+import java.util.Optional;
+
+import org.jabref.logic.bibtex.FieldWriter;
+
+public enum Langid {
```

Comment:
Please reference the biblatex manual as source of these values. You can use `<a href ..`) for JAvaDoc or Markdown syntax (with three ///)

Reply:
Can these values also be used for the `language` field?